configure

```diff
@@ -1057,7 +1057,7 @@ case "$arch,$system" in
                   aspp="${TOOLPREF}cc -c";;
   *,freebsd)      as="${TOOLPREF}as"
                   aspp="${TOOLPREF}cc -c";;
-  amd64,*|arm,*|arm64,*|i386,*|power,bsd*,*)
```

Comment:
I was confused about the presence of this ",*" in the first place -- was that just a mistake?

Reply:
Yes, I think this was simply a mistake, since we're not matching against a triplet but a tuple here.